features/promotion/managing_promotions/adding_promotion_with_action.feature

```diff
@@ -10,8 +10,8 @@ Feature: Adding a new promotion with action
 
     @ui @javascript
     Scenario: Adding a new promotion with fixed discount
-        Given I want to create a new promotion
-        When I specify its code as "10_for_all_products"
+        When I want to create a new promotion
+        And I specify its code as "10_for_all_products"
```

Comment:
So IMO these steps should looks like `Given I am creating a new promotion` or sth like that. Now we are extremely incoherent with naming steps like these and I guess that we should decide on some convention.